### ISSUE TITLE
Prepare initial FY'17 planning and estimates

### ISSUE THREAD

**Issue by gbinal** [18F/ED-API-Program#73]

**Comment by metzgr** [comment]
Here are our estimates for FY16: 
![image](https://cloud.githubusercontent.com/assets/18745563/16305677/df286256-3928-11e6-8d1e-a86012ac612f.png)

A supervisor asked us to find a more precise estimate of hosting expenses, since the funding source we're using to pay for this work is pretty critical to office operations and we shouldn't overcommit funds. We connected with Diego Lapiduz (met him in a Scorecard conversation) who told us 10K should cover a year's of hosting for 2-3 APIs.

Basically, we want to pay for the labor under this IAA, cloud.gov access, and a year's worth of hosting all in FY16.

**Comment by gbinal** [comment]
Questions to answer: 
- Can we prebuy the FY'17 access package in this FY'16.  
- Can we pre-commit money for usage costs?  
- Can we do this in the current IAA?  
- Do we need to commit further funds?  
- Are there labor needs/plans for FY'17?

**Comment by metzgr** [comment]
Diego's comments:
- Because of how our finance works we can set up an IAA with FY16 money and bill it monthly entering the FY17. We can only do that within 1 calendar year of the IAA being signed though so if we sign the IAA in July then the PoP will end in July/17.
- It really depends on how many APIs you host, what they do and what services they consume. The more you use it the more money it will take. I think that 5 - 10k/yr should cover 1-3 APIs.
- The base price is $20k / yr and on top of that you need to add usage. If you are running only a small to medium application your usage will be in the order of $5-10k/yr.

If you want to hedge on the IAA (usage costs are severable) 20k/yr for usage buys you a good amount of headroom.

From cloud.gov:
- Resource usage
  To support tenant applications and managed service instances, cloud.gov allocates resources (like compute nodes and memory) from an underlying IaaS. What we charge for this component covers the costs for the resources that cloud.gov allocates to hosted systems and teams. It also includes a small margin that covers our labor in managing those resources.

Resource usage fees on-demand are understood to be “severable” as they can be controlled in a self-service manner by provisioning more or fewer apps, and therefore must be treated accordingly.

**Agencies can purchase reserved capacity via 18F as well**, which is understood to be “non-severable”.

**Comment by gbinal** [comment]
This has been accomplished in later efforts.
